Sources/View/NibOwnerLoadable.swift

```diff
@@ -38,27 +38,25 @@ public extension NibOwnerLoadable where Self: UIView {
 
    - parameter owner: The instance of the view which will be your File's Owner
                       (and to which you want to add the XIB's views as subviews).
-                      Defaults to a brand new instance if not provided.
-   - returns: A `NibOwnLoadable`, `UIView` instance
+   - returns: A `NibOwnerLoadable`, `UIView` instance
    */
   @discardableResult
-  static func loadFromNib(owner: Self = Self()) -> Self {
```

Comment:
I was wondering about that too. I don't know if that's github bug, however, it made me noticed that documentation comments needs to be changed 😄